### ISSUE TITLE
Help - Is it possible to incorporate markdown tables of multiple UTM geographic coordinates in Obsidian MapView plugin?

### ISSUE THREAD

**Issue by jcmoral90** [esm7/obsidian-map-view#52]
I have a markdown table with approx. 500 UTM coordinates. Is it possible to transfer all points to Map View? If so, where can I access the documentation.

Thanks

**Comment by esm7** [comment]
You can definitely do that but it will require a script or another method of bulk text editing to:
1. Convert the UTM to "geographical" (lat, lng) coordinates (the [formulas](https://en.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system#From_UTM_coordinates_(E,_N,_Zone,_Hemi)_to_latitude,_longitude_(%CF%86,_%CE%BB)) are quite complicated, you'd probably want to use something pre-made).
2. Convert the coordinates to geolocation links in a standard format i.e. convert a point (x,y) to `[](geo:x,y)`.

You can find online convertors ([here](https://www.engineeringtoolbox.com/utm-latitude-longitude-d_1370.html) for example) or find templates for shreadsheets like Excel/GDocs.
In summary: very possible but requires some effort.
